data/files/src/main/java/tech/pegasys/teku/data/files/SyncDataAccessor.java

```diff
@@ -11,7 +11,7 @@
  * specific language governing permissions and limitations under the License.
  */
 
-package tech.pegasys.teku.core.signatures;
+package tech.pegasys.teku.data.files;
```

Comment:
Since we're creating a new module (which is a good idea), I'd call it `:infrastructure:io` rather than `:data:files`.